crates/rover-client/src/query/graph/check.graphql

```diff
@@ -0,0 +1,35 @@
+mutation CheckSchemaQuery(
+    $graphId: ID!
+    $variant: String
+    $schema: String
+  ) {
+    service(id: $graphId) {
+      checkSchema(
+        proposedSchemaDocument: $schema
+        baseSchemaTag: $variant
+      ) {
+        targetUrl
+        diffToPrevious {
+          severity
+          affectedClients {
```

Comment:
I think we stopped supporting this field and just return empty arrays now, so we might want to remove it from the query.